examples/tutorials/self-paced-training/part-3_security_and_privacy/chapter-6_Security_in_federated_compute_system/06.3_site_security_privacy_policy/site_policy.ipynb

```diff
@@ -0,0 +1,475 @@
+    "* can perform shell commands ( see job category for details) from the site of the same organization \n",
+    "* can not have byoc (bring your own code): customized code in the \"custom\" directory. \n",
+    "\n",
+    "Similarly we have other user role policies\n",
+    "\n",
+    "\n",
+    "For the **lead** user\n",
+    "\n",
+    "| Capability       | Permission Scope |\n",
+    "|------------------|------------------|\n",
+    "| submit_job       | o:site           |\n",
+    "| clone_job        | n:submitter      |\n",
+    "| manage_job       | n:submitter      |\n",
+    "| download_job     | n:submitter      |\n",
+    "| view             | any              |\n",
+    "| operate          | o:site           |\n",
+    "| shell_commands   | o:site           |\n",
+    "| byoc             | any              |\n",
+    "\n",
+    "The user \"lead\" is a \"submitter\" that can but only can submit to its organizational own site (considering one organization has many sites)\n",
+    "\n",
+    "\n",
+    "\n",
+    "For the **member** user\n",
+    "\n",
+    "| Capability | Permission Scope |\n",
+    "|------------|------------------|\n",
+    "| view       | any              |\n",
+    "\n",
+    "    "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "deb08e63",
+   "metadata": {},
+   "source": [
+    "We can take a look at the policies for \n",
+    "\n",
+    "##### Site_b privacy policy"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "08461d9a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!cat {Workspace}/site_b/local/privacy.json"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "319eb7a5",
+   "metadata": {},
+   "source": [
+    "| scope   | default_scope | task_data_filters                       | task_result_filters                                                |\n",
+    "|---------|----------------|-----------------------------------------|--------------------------------------------------------------------|\n",
+    "| public  | public         |                                         |                                                                    |\n",
+    "| test    | public         | test_filter.TestFilter                  | test_filter.TestFilter                                             |\n",
+    "| private | public         | test_filter.TestFilter                  | nvflare.app_common.filters.percentile_privacy.PercentilePrivacy    |\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7c84b47b",
+   "metadata": {},
+   "source": [
+    "In the privacy policy, we noticed that there are different scopes can be defined. The scope-specified policy is enforced the the FLARE's filter mechanism. For each non-public scope, one can define the different filters to enforce the desired behavior.  For example, the organization administrator can instrument filter to prevent certain type of data leakage accidentally exposed by data sicentists\n",
```

Comment:
"In the privacy policy, we noticed that different scopes can be defined. The scope-specific policy is enforced by FLARE's filter mechanism. For each non-public scope, different filters can be defined to enforce the desired behavior. For example, the organization administrator can implement filters to prevent certain types of data leakage that may be accidentally exposed by data scientists."